### PR TITLE
Pull revised shooter into master

### PR DESCRIPTION
**Pull request by cannedlock**

### CODE REVIEW

File: components/ShooterLogic.py [hunk 1]
```diff
@@ -18,20 +18,20 @@ def __init__(self):
     def runLoaderManually(self):
         self.isAutomatic = False
 
-    def getAutomaticStatus(self):
-        return self.isAutomatic
-
     def stopManual(self):
         self.isAutomatic = True
 
+    def getAutomaticStatus(self):
```
Comment (ChrisHirsch) [inline components/ShooterLogic.py]: Can we start putting comments in the functions please? ie what it's supposed to do etc

File: components/ShooterLogic.py [hunk 2]
```diff
@@ -18,20 +18,20 @@ def __init__(self):
...
+
     def fireShooter(self):
         if not self.isAutomatic:
             self.shooterMotors.stopIntake()
             self.shooterMotors.stopLoader()
             print("manual shooter running")
             self.shooterMotors.runShooter(1)
             if self.shooterMotors.shooterMotor.getEncoder().getVelocity() >= 5000:
-                self.shooterMotors.runLoader(1 * self.loaderMulti)
+                self.shooterMotors.runLoader(0.6 * self.loaderMulti)
```
Comment (ChrisHirsch) [inline components/ShooterLogic.py]: can we call loaderMulti something more definitive?

File: components/ShooterLogic.py [hunk 3]
```diff
@@ -87,12 +87,12 @@ def __init__(self):
     def runLoaderAutomatically(self):
         self.isAutomatic = True
 
-    def getAutomaticStatus(self):
-        return self.isAutomatic
-    
     def stopAutomatic(self):
         self.isAutomatic = False
 
+    def getAutomaticStatus(self):
```
Comment (ChrisHirsch) [inline components/ShooterLogic.py]: Comment on what "automatic status is"

File: components/ShooterLogic.py (hunk 2 above)
Comment (ChrisHirsch) [inline components/ShooterLogic.py]: Can this 0.6 be a tunable with a default?

File: components/ShooterLogic.py [hunk 4]
```diff
@@ -162,9 +162,9 @@ def beginLoading(self):
         else:
             self.logicArray = []
 
-    def switchToReverse(self):
+    def initAutoShooting(self):
         if self.isAutomatic:
-            if self.SensorArray[0].get() == False and self.shooterMotors.isLoaderActive() == False:
+            if not self.SensorArray[0].get() and not self.shooterMotors.isLoaderActive():
```
Comment (ChrisHirsch) [inline components/ShooterLogic.py]: Should we be using enums insteadl of [0] and maybe a comment of what this is doing exactly?

Comment (cannedlock) [reply components/ShooterLogic.py]: added comment

File: components/ShooterLogic.py (hunk 2 above)
Comment (cannedlock) [reply components/ShooterLogic.py]: changed loader values to tunables

File: components/ShooterLogic.py (hunk 3 above)
Comment (cannedlock) [reply components/ShooterLogic.py]: added comment

File: components/ShooterLogic.py (hunk 2 above)
Comment (cannedlock) [reply components/ShooterLogic.py]: changed to self.loaderMotorVal

File: components/ShooterLogic.py (hunk 1 above)
Comment (cannedlock) [reply components/ShooterLogic.py]: added some comments on these

**Pull request merged**